bin/plugins.json

```diff
@@ -189,15 +189,15 @@
     "require_install": "true",
     "category": "client"
   },
-  "perf-top": {
+  "perftop": {
```

Comment:
I see perf-top also has Macos, Linux zip. Do we want to check availability for both?

Reply:
1. no need as they both belongs to zip version.
2. even we want to check the entire regex needs to be refactored as the postfix of MACOS and LINUX is after the version number so the current setup wont identify which is which for now.